cristin-import/src/main/java/no/unit/nva/cristin/mapper/CristinMapper.java

```diff
@@ -243,11 +243,14 @@ private AdditionalIdentifier extractIdentifier() {
     }
 
     private String extractNpiSubjectHeading() {
-        if (!isBook(cristinObject)) {
+        if (!isBook(cristinObject) && !isReport(cristinObject)) {
```

Comment:
the code is not easily readable. Code should be readable as normal text as much as possible.
```
  private String extractNpiSubjectHeading() {
        CristinSubjectField subjectField = extractSubjectField();
        if (isNull(subjectField)) {
            return EMPTY_STRING;
        } else {
            return extractSubjectFieldCode(subjectField);
        }
    }

    private String extractSubjectFieldCode(CristinSubjectField subjectField) {
        return Optional.ofNullable(subjectField.getSubjectFieldCode())
            .map(String::valueOf)
            .orElseThrow(() -> new MissingFieldsException(CristinSubjectField.MISSING_SUBJECT_FIELD_CODE));
    }

    private boolean resourceShouldAlwaysHaveAnNpiSubjectHeading() {
        return cristinObject.getSecondaryCategory().equals(CristinSecondaryCategory.MONOGRAPH);
    }

    private boolean resourceTypeIsNotExpectedToHaveAnNpiSubjectHeading() {
        return !(isBook(cristinObject) || isReport(cristinObject));
    }

    private CristinSubjectField extractSubjectField() {
        if (resourceTypeIsNotExpectedToHaveAnNpiSubjectHeading()) {
            return null;
        }
        CristinSubjectField subjectField = extractCristinBookReport().getSubjectField();
        if (resourceShouldAlwaysHaveAnNpiSubjectHeading() && isNull(subjectField)) {
            throw new MissingFieldsException(CristinBookOrReportMetadata.SUBJECT_FIELD_IS_A_REQUIRED_FIELD);
        }
        return subjectField;
    }
```